assets/src/components/button/style.scss

```diff
@@ -0,0 +1,83 @@
+/**
+ * Button styles,
+ */
+
+
+.newspack-button {
```

Comment:
As per the changes made in https://github.com/Automattic/newspack-plugin/pull/47, let's change the className to `muriel-button`. We'll need to make this change in the component file and the tests as well as here.